src/SFA.DAS.Apprenticeships.Domain/Apprenticeships/Api/PendingPriceChange.cs

```diff
@@ -27,17 +28,12 @@ public static class PendingPriceChangeExtensions
 	{
 		public static InitiatedBy GetPriceChangeInitiatedBy(this PendingPriceChange pendingPriceChange)
 		{
-			if(pendingPriceChange.ProviderApprovedDate.HasValue && !pendingPriceChange.EmployerApprovedDate.HasValue) 
+			if(Enum.TryParse<InitiatedBy>(pendingPriceChange.Initiator, out var initiatedBy)) 
```

Comment:
can we change initiatedBy to initiator to match everywhere else?

Reply:
Done